### DISCUSSION TITLE
Why doesn't RabbitMQ RPM have an Enterprise Linux 9 (.el9) variant?

### DISCUSSION THREAD

**Discussion by vlad88sv** [rabbitmq/rabbitmq-website#1907] [General]
### Is your feature request related to a problem? Please describe.

I just noticed the el/9 URL offers .el8 - is this intentional or is this going to change to .el9 packages in the future?

### Describe the solution you'd like

.el9 packages for Rocky Linux 9

### Describe alternatives you've considered

Using current .el8 packages on EL9, it just feels like it may be wrong.

**Comment by michaelklishin** [comment]
There never were RabbitMQ RPM packages specifically for "el9" because they are not necessary: it's an architecture-neutral package without native code. The same applies to our Debian package, it is a single package used on more than ten distributions, assuming that a supported version of Erlang is provisioned. You can inspect them at [rabbitmq/rabbitmq-packaging](https://github.com/rabbitmq/rabbitmq-packaging).

Zero dependency Erlang RPMs produced by our team [do provide "el9" packages](https://github.com/rabbitmq/erlang-rpm/releases/) and it has been the case for years.
